### PR TITLE
Mopub 5.8.0

### PR DESCRIPTION
**Pull request by ariedov**
Using the newest `.aar` files for mopub 5.8 + new binding project for MoPub Volley - since the errors there were hidden when it was just a `ReferenceJar`.

### CODE REVIEW

**Comment by ariedov** [comment]
[Update Xamarin Android MoPub SDK](https://trello.com/c/TiWVzBOu/1220-update-xamarin-android-mopub-sdk)

File: .idea/.idea.mopub-android-xamarin/.idea/contentModel.xml [hunk 1]
```diff
@@ -0,0 +1,389 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (michd) [inline .idea/.idea.mopub-android-xamarin/.idea/contentModel.xml]: I'm guessing `.idea/` probably shouldn't be checked it; I recommend adding it to .gitignore.

File: mopub-sdk-base-android-binding/Transforms/Metadata.xml [hunk 2]
```diff
@@ -21,32 +21,74 @@
     <method name="remove" return="void" abstract="false" native="false" synchronized="false" static="false" final="false" deprecated="not deprecated" visibility="public">
     </method>
   </add-node>
-  
+
+  <add-node path="/api/package[@name='com.mopub.common.privacy']/class[@name='SyncRequest']">
+    <method name="compareTo" return="int" abstract="false" native="false" synchronized="false" static="false" final="false" deprecated="not deprecated" visibility="public">
+      <parameter name="o" type="java.lang.Object" />
+    </method>
+  </add-node>
```
Comment (michd) [inline mopub-sdk-base-android-binding/Transforms/Metadata.xml:29]: I won't pretend I know exactly what these `add-node`s are doing... My guess would be adding to the API something that wasn't there from the automatic binding? Can you maybe elaborate on what these are doing?

**Review by michd — CHANGES_REQUESTED** [review]
Mainly requesting changes to `.gitignore` `.idea/` here, other than that everything looks fine to me. :+1:

File: mopub-sdk-base-android-binding/Transforms/Metadata.xml (hunk 2 above)
Comment (ariedov) [reply mopub-sdk-base-android-binding/Transforms/Metadata.xml:29]: I will not pretend either, the documentation is super vague on this one. I just had the exact same issue with `compareTo` before, used the same solution as before and it worked.

**Comment by ariedov** [comment]
`.idea` folder removed

**Review by michd — APPROVED** [review]
Tentative g2g, but I guess it'll need changes to address the crashes found... :/

**Comment by SimpsOff** [comment]
G2G with updated Xamarin.Android from the build server. Also passed QA checks.

**Pull request merged**